manage-data/migrate/migrate-data-between-elasticsearch-clusters-with-minimal-downtime.md

```diff
@@ -0,0 +1,77 @@
+* Ensure data ingestion, such as {{ls}} or {{beats}}, and data consumption, such as applications using {{es}} as a backend, seamlessly migrate to the new cluster.
+* Maintain data consistency and minimize disruption.  
+
+## How incremental snapshots works [how-incremental-snapshots-work]
+Incremental snapshots capture only the data that has changed since the previous snapshot. 
+
+After the initial full snapshot, each subsequent snapshot contains only the differences, which makes the snapshot process faster over time. When you restore snapshots, only the missing data segments are copied from the snapshot repository to the cluster local storage, speeding up restores when the changes between snapshots are small.
+
+When you incrementally create and restore snapshots, you can repeatedly synchronize the new cluster with the old cluster by taking and restoring multiple snapshots before performing the final cutover.
+
+## Recommended migration timeline [recommended-migration-timeline]
+Complete the minimal-downtime migration using incremental snapshots. While the exact sequence may differ depending on your infrastructure and operational requirements, you can use the recommended migration timeline as a reliable baseline that you can adapt. Adjust the steps and times to fit your own operational needs.
+
+1. **09:00**: Take the initial full snapshot of the old cluster. You can also take the initial full snapshot the day before.
+2. **09:30**: Restore the snapshot to the new cluster.
+3. **09:55**: Take another snapshot of the old cluster and restore it to the new cluster. Repeat this process until the snapshot and restore operations take only a few seconds or minutes.
+4. **10:15**: Perform the final cutover.
+    1. In the old cluster, pause indexing or set indices to ready-only.
+    2. Take a final snapshot. 
+    3. Restore the snapshot to the new cluster. 
+    4. Change ingestion and querying to the new cluster. 
+    5. Open the indices in the new cluster. 
+
+## Incremental snapshot limitations [incremental-snapshot-limitations]
+While incremental snapshots allow efficient migration with minimal downtime, consider the limitations when planning your migration.
+
+Limitations include the following:
+* **Storage requirements** – Sufficient repository storage is required, and usage can grow based on snapshot frequency and data volume.
+* **Network overhead** – Transferring snapshots across networks, regions, or providers can be time-consuming and incur costs.
+* **Version compatibility** – Old and new clusters must use compatible {{es}} versions. To check if your cluster versions are compatible, check [Snapshot compatibility](/deploy-manage/tools/snapshot-and-restore.md#snapshot-restore-version-compatibility).
+* **Custom integrations** – Some custom integrations that directly use the {{es}} API can require additional handling during the cutover from the old cluster to the new cluster.
```

Comment:
do we have a source or guidance for this?

Reply:
Maybe a reference to e.g. the Elasticsearch Java client library?